.github/workflows/integration-test.yml

```diff
@@ -8,11 +8,22 @@ on:
 
 jobs:
 
-  build:
-
+  integration-tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build_type: [local-data, remote-data]
 
     steps:
     - uses: actions/checkout@v2
+    - name: Download data
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
some configurations here, you might not need all of these for downloading your resources.